src/logging.ts

```diff
@@ -0,0 +1,13 @@
+import { Logger } from 'tslog';
+
+/**
+ * LOG is the SDK's base logger
+ */
+export const LOG = new Logger();
+
+/**
+ * setLogLevel adjusts the minimum level of logs which are output by the SDK's logger
+ */
+export function setLogLevel(level: number) {
```

Comment:
Is this somehow being called implicitly somewhere?

Reply:
Reading your PR description more carefully I see the purpose now. Perhaps we have enough static checks to see that this function is well formed. Regardless, a unit test can act as a sanity check.

Reply:
It's not being called anywhere, it's set that users can set the log level to turn off logs if they don't want to see a certain level.